src/wazuh_qa_framework/system/wazuh_handler.py

```diff
@@ -843,3 +844,104 @@ def is_manager(self, host):
             bool: True if host is manager
         """
         return host in self.get_managers()
+
+    def create_group(self, group_name):
```

Comment:
We should also have the [API method](https://documentation.wazuh.com/current/user-manual/api/reference.html#operation/api.controllers.agent_controller.post_group)

```suggestion
    def create_group(self, group_name, method):
```

Reply:
Done in https://github.com/wazuh/qa-system-framework/pull/71/commits/6e4fa8a41bfb8248e339d6eb0911ce229ca57084